components/actionbar/definitions.css

```diff
@@ -0,0 +1,55 @@
+/* stylelint-disable spectrum-tools/no-unused-custom-properties -- definitions file */
```

Comment:
@castastrophe do we want this file to compile out as a `dist` artifact? I could see arguments for or against it. Happy to do whatever we think makes the most sense.